### ISSUE TITLE
Spawning is broken 

### ISSUE THREAD

**Issue by ribbybibby** [ribbybibby/Grimm-Journey#3]
The spawn mechanism doesn't seem to work in the submitted LD build. I believe this is because the spawn point objects weren't linked to the spawn controller in most of the levels. 

I've fixed this in the first level and it seems to do the trick but I have to do all the existing levels and also have a look at any possible improvements in the spawn code.

**Comment by ribbybibby** [comment]
When you launch the first level from the editor, you get one 'missing script' error in the console. If you launch it from the main menu you get two.

I'm not sure what the first one is, the editor should highlight it but it doesn't, nevertheless I think that the level transition is somehow destroying the spawn controller because I attached logging to it that ceases if you launch from main menu.

Still plodding away with this one.

**Comment by ribbybibby** [comment]
^ the above turned out not to be true. It was the BBW2 prefab that was missing a script (fixed this now).

There might be something in the SpawnController script that depends on something the level transition code skips over.

**Comment by ribbybibby** [comment]
Fixed in 12cbc887b37e30162f8c96bd27ba1aa7213cfc6f

FINALLY figured this out after stepping through the SpawnController script step by step with logging. 

We spaced the proc of each spawn out by generating a timer value which would be the current time + an arbitrary amount. Then, we would run our spawn code when timer value == current time. On first run, this worked fine, however, for an as yet unknown reason, it didn't work after a level transition.

What would seem to happen is that Time.time would reach the value of the timer but then go past it without procing the spawn. I can only assume that the window was so ridiculously tight that something about the level transition put it off.

Solution:
Change == to >=, so that even a Time.time value above the timer will proc the spawn. This seems to work without any side effects. It should have been this way from the start (me = noob).
